exporter/stackdriverexporter/factory.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriverexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
```

Comment:
Let's follow the same ordering rule we have in the core:
1. Standard packages
2. 3rd party dependencies
3. Our dependencies
Separate sections from each other by a blank line.

Reply:
SG, done.